numpy/testing/utils.py

```diff
@@ -903,12 +908,12 @@ def compare(x, y):
         # casting of x later.
         dtype = result_type(y, 1.)
         y = array(y, dtype=dtype, copy=False, subok=True)
-        z = abs(x-y)
+        z = abs(x - y)
 
         if not issubdtype(z.dtype, number):
             z = z.astype(float_)  # handle object arrays
 
-        return around(z, decimal) <= 10.0**(-decimal)
+        return z < 1.5 * 10.0**(-decimal)
```

Comment:
Yeah need a test. If there had been one in the first place we would never have ended up with `1.5 * 10**(-decimal)`. That was due to the use of a `<=` instead of `<`, and I think that came about from working around some other perceived bug. I'd just change it if it didn't cause so many new failures in downstream code.